client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -11,12 +11,12 @@ import ClipboardButton from 'calypso/components/forms/clipboard-button';
 import { encodePatternId } from 'calypso/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { PatternsGetAccessModal } from 'calypso/my-sites/patterns/components/get-access-modal';
+import { PatternTypeFilter } from 'calypso/my-sites/patterns/types';
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import getUserSetting from 'calypso/state/selectors/get-user-setting';
 import { getTracksPatternType } from '../../lib/get-tracks-pattern-type';
-import type {
-	Pattern,
-	PatternGalleryProps,
-	PatternTypeFilter,
-} from 'calypso/my-sites/patterns/types';
+import type { Pattern, PatternGalleryProps } from 'calypso/my-sites/patterns/types';
```

Comment:
```suggestion
import { PatternTypeFilter, type Pattern, type PatternGalleryProps } from 'calypso/my-sites/patterns/types';
```
Nit, but we can import all modules from `calypso/my-sites/patterns/types` in a single statement, even if they're not all types. 

In fact, we probably don't need to `import type` instead of just `import`. I usually do that out of old habit, but judging by a quick Google search, it doesn't seem to be necessary (see, e.g. [this](https://stackoverflow.com/a/66833435/618739))